### DISCUSSION TITLE
Desktop Set Up

### DISCUSSION THREAD

**Discussion by DovarFalcone** [matthiasn/lotti#1397] [Q&A]
Are there instructions on how to set the desktop version up on Windows somewhere I can reference?

**Comment by matthiasn** [comment]
@DovarFalcone apologies for the delay, didn't see a notification. If you have a running Flutter development environment, you can run `flutter run -d windows`. Please also check the project README about Windows releases, which I just updated.

**Comment by matthiasn** [comment]
GitHub Discussions hasn't become a part of my regular workflow yet, but maybe I should create a habit for that 🤓

**Comment by corvusdeinanis** [comment]
Also a quick dirty way to install the msix if Windows is not letting you (I have developer mode on, I've used a self-signed certificate, yada yada) is to extract the msix with something like msix hero (https://github.com/marcinotorowski/MSIX-Hero). 

Probably not the best way, but if you're in a pinch and all else fails :D

**Reply by matthiasn** [reply]
@corvusdeinanis thanks, I was missing that part when trying (and failing) to install the unsigned app.

**Reply by corvusdeinanis** [reply]
Cheers! Love the app, thank you for it :)

**Comment by xuesheng88** [comment]
Hi. Would someone be willing to help me? I have a Linux desktop, running Pop OS. And, I can't get Whisper A.I. to function properly. I don't see the tick boxes like in the video Matthiasn showcased. I downloaded Whisper A.I. via the terminal, and I can transcribe there, but to no success can I get it operate via Lotti. I would like this to work, if possible.
<img width="1344" height="1358" alt="ksnip_20260111-035335" src="https://github.com/user-attachments/assets/6b5027b3-4ae3-4baf-ab3e-c798ecb5828c" />

**Reply by matthiasn** [reply]
Hi @xuesheng88, thanks for the question. You will have to make the audio transcription prompt available to the respective category that you are using, otherwise the checkboxes won't be available. I'm about to create a video showing the setup for doing transcription using Google's Gemini 3 in the cloud, which is the easiest way to get transcription to work. Will post the link here when done, then the next one should be about the (somewhat more involved) local setup.

**Reply by matthiasn** [reply]
@xuesheng88 you can find the recording here: https://matthiasnehlsen.substack.com/p/ftue-improvements-on-linux, where I explain the categories and whisper local setup as well, in addition to the improved initial setup on Linux. Please let me know if that helps. Thanks

**Reply by xuesheng88** [reply]
Okay. Thank you very much for your reply. I sincerely appreciate it. And, thank you for your dedication in creating such a helpful tool. I expect that you'll be able to have successful longevity with this project.
